packages/strapi-design-system/src/Avatar/Avatar.js

```diff
@@ -1,21 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
-const avatarSize = '26px';
+const avatarSize = 26;
+const previewSize = 64;
 
 const AvatarImg = styled.img`
   border-radius: 50%;
   display: block;
-  height: ${avatarSize};
-  width: ${avatarSize};
+  position: relative;
 `;
 
-export const Avatar = ({ src, alt }) => {
-  return <AvatarImg src={src} alt={alt} width={avatarSize} height={avatarSize} />;
+const AvatarImgWrapper = styled.div`
+  position: relative;
+`;
+
+const PreviewContainer = styled.img`
+  border-radius: 50%;
+  position: absolute;
+  transform: translate(-${(previewSize - avatarSize) / 2}px, -100%);
+  margin-top: -${({ theme }) => theme.spaces[1]};
+`;
+
+const Overlay = styled.div`
+  z-index: 1;
+  border-radius: 50%;
+  position: absolute;
+  width: ${avatarSize}px;
+  height: ${avatarSize}px;
+  background: ${({ theme }) => theme.colors.neutral0};
+  opacity: 0.4;
+`;
+
+export const Avatar = ({ src, alt, preview }) => {
+  const [previewVisible, setPreviewVisible] = useState(false);
+
+  return (
+    <div>
+      {preview && previewVisible ? (
+        <PreviewContainer
+          aria-hidden
+          alt=""
+          width={`${previewSize}px`}
+          height={`${previewSize}px`}
+          src={preview === true ? src : preview}
+        />
+      ) : null}
+
+      <AvatarImgWrapper onMouseEnter={() => setPreviewVisible(true)} onMouseLeave={() => setPreviewVisible(false)}>
```

Comment:
Should we change this wrapper to an inline element so that the mouse over will be triggered only on the AvatarImg and not on the div row 🤔  ?

![avatar](https://user-images.githubusercontent.com/7756284/126614908-be3219b6-6d8a-45bb-bb73-0b9314748fcc.gif)

Reply:
Also, should we handle the focus on that component so we can trigger the zoom with the keyboard?

Reply:
I'm wondering about the impact of this? The image is already so small 🤔 . Do you think we could benefit from this?

Reply:
We can keep it like that and see if there is any feedback on the use 👍🏽